meta/style-guide/README.md

```diff
@@ -0,0 +1,171 @@
+# Evergreen Style Guide
+
+This document serves to provide a set of guidelines for how to write good
+documentation for Evergreen. This guide is less focused on the content and
+more focused on the style and structure of the documentation.
+
+## People
+
+Whenever referring to someone in Evergreen, if they are a member of our
+team and have an entry in [purduehackers/dark-forest](https://github.com/purduehackers/dark-forest)
+then you should ALWAYS use the following format to refer to them:
+
+✅ Correct
+
+```markdown
+[@rayhanadev](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/rayhanadev.md)
```

Comment:
it's not abundantly clear from this where i get this username. should mention explicitly that the username should be pulled from dark-forest